### ISSUE TITLE
Mobile Web Browser: Emoji picker is not in viewport when opened (Iphone)

### ISSUE THREAD

**Issue by kkovrizhenko** [RocketChat/Rocket.Chat#11128]
### Steps to reproduce:

1. Go to Rocket Chat using Mobile web browser (or simulator fo relevant screen size)
2. Open any DM or chat room
3. Click on Emoji icon in the input area

### Expected behavior:

Emoji picker is opened

### Actual behavior:
Emoji picker is not visible, hidden under left side of the screen
![2018-06-13_1632](https://user-images.githubusercontent.com/9197041/41354359-7bd7de02-6f27-11e8-9634-6d76c1c0d3d9.png)

**Comment by ramrami** [comment]
I have the same problem. 
I think it's because of this condition in `rocketchat-emoji/client/lib/EmojiPicker.js`: 

```javascript
		if (left + this.width >= window.innerWidth) {
			cssProperties.left = left - this.width - windowBorder;
		}
```

**Comment by ramrami** [comment]
@zigfighter @reetp  I just opened a fix for this problem #12457

**Comment by zigfighter** [comment]
@ramrami looking good ! needs to be reviewed
